iOS/DuckDuckGo/DownloadManager.swift

```diff
@@ -259,3 +225,9 @@ extension NSNotification.Name {
     static let downloadFinished: NSNotification.Name = Notification.Name(rawValue: "com.duckduckgo.notification.downloadFinished")
     static let downloadsDirectoryChanged: NSNotification.Name = Notification.Name(rawValue: "com.duckduckgo.notification.downloadsDirectoryChanged")
 }
+
+
+extension DownloadManager {
```

Comment:
Forwarding expected properties from `DownloadManager` now that `DownloadsDirectoryHandling` is its own protocol